### PR TITLE
Add gem simple calendar, setup styling template and setup routes

### PR DESCRIPTION
**Pull request by LauraRe**
#### Description:
Added a calendar on the landing page for laundry booking.

Changes proposed in this pull request:
* Created Booking controller
* Installed simple_calendar

#### What I have learned working on this feature: 
Learned how to use simple_calendar gem

#### Screenshots:
image

### CODE REVIEW

**Review by Joberg95 — COMMENTED** [review]
Not sure if we should keep the css classes, since we are not using them.

File: app/controllers/calendar_controller.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class CalendarController < ApplicationController
```
Comment (magnus-thor) [inline app/controllers/calendar_controller.rb]: I dont think Calendar is a good name for  the controller, choose something more descriptive of what it is controlling, ie what will you be using it for?  crud on bookings? crud on laundry rooms?

**Review by magnus-thor — CHANGES_REQUESTED** [review]

File: app/controllers/calendar_controller.rb (hunk 1 above)
Comment (LauraRe) [reply app/controllers/calendar_controller.rb]: changed controller's name to Booking.

**Review by magnus-thor — CHANGES_REQUESTED** [review]
I cant believe I didn't notice it the first time, but you have no feature file here, why is that?

**Review by amandagibson — APPROVED** [review]

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -1,3 +1,5 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+  def index
```
Comment (magnus-thor) [inline app/controllers/application_controller.rb]: Remove the index method from the application controller

**Review by magnus-thor — CHANGES_REQUESTED** [review]

**Review by magnus-thor — APPROVED** [review]

**Pull request merged**